src/themes/custom_theme/custom_theme.js

```diff
@@ -0,0 +1,246 @@
+/**
+ * Created by apoorv on 9/06/16.
+ */
+define(['jquery', 'windows/windows', 'charts/charts', 'highstock', 'color-picker', 'jquery-growl'], function($, windows, charts){
+	
+	var chartData = [],
+			win = null,
+			themeConf = {};
+
+	function createThemeObject(id, value, alpha){
+		if(!value)
+			return;
+
+		var rgba = [];
+		rgba[0] = parseInt(value.substring(0,2),16),
+		rgba[1] = parseInt(value.substring(2,4),16),
+		rgba[2] = parseInt(value.substring(4,6),16),
+		rgba[3] = alpha? parseFloat(alpha) : 1;
+		var temp = "",
+				end = "";
+		var propertyNames = id.split("_");
+		propertyNames.forEach(function(property, index, arr){
+			// For creating array.
+			if(property === "arr"){
+				temp = temp + "[" + "\"rgba(" + rgba.join(", ") + ")\"" + "]";
+				return;
+			}
+			end = end + "}";
+			if(index === arr.length - 1){
+				temp = temp + "{" + "\"" + property + "\"" + ":" + "\"rgba(" + rgba.join(", ") + ")\"";
+				return;
+			}
+			temp = temp + "{" + "\"" + property + "\"" + ":";
+		});
+		temp = temp + end;
+		$.extend(themeConf, JSON.parse(temp));
+	}
+
+	$('a.theme_custom')
```

Comment:
@4p00rv This is incorrect. Please refer to download.js and navigator.js how it is getting loaded on click of the menu item. We should load modules when users want it